napi.h

```diff
@@ -789,36 +841,31 @@ namespace Napi {
     napi_property_descriptor _desc;
   };
 
-  /*
-   * Base class to be extended by C++ classes exposed to JavaScript.
-   * Each C++ class instance gets "wrapped" by a JavaScript object.
-   *
-   * At initialization time, the DefineClass() method must be used to
-   * hook up the accessor and method callbacks. It takes a list of
-   * property descriptors, which can be constructed via the various
-   * static methods on the base class.
-   *
-   * Example:
-   *
-   *   class Example: public Napi::ObjectWrap<Example> {
-   *   public:
-   *     static void Initialize(Napi::Env& env, Napi::Object& target) {
-   *       Napi::Function constructor = DefineClass(env, "Example", New, {
-   *         InstanceAccessor("value", &GetValue, &SetValue),
-   *         InstanceMethod("doSomething", &DoSomething),
-   *       });
-   *       target.Set("Example", constructor);
-   *     }
-   *
-   *     static Example* New(const Napi::CallbackInfo& info) {
-   *       return new Example();
-   *     }
-   *
-   *     Napi::Value GetValue(const Napi::CallbackInfo& info);
-   *     void SetValue(const Napi::CallbackInfo& info, const Napi::Value& value);
-   *     Napi::Value DoSomething(const Napi::CallbackInfo& info);
-   *   }
-   */
+  /// Base class to be extended by C++ classes exposed to JavaScript; each C++ class instance gets
+  /// "wrapped" by a JavaScript object that is managed by this class.
+  ///
+  /// At initialization time, the `DefineClass()` method must be used to
+  /// hook up the accessor and method callbacks. It takes a list of
+  /// property descriptors, which can be constructed via the various
+  /// static methods on the base class.
+  ///
+  /// #### Example:
+  ///
+  ///     class Example: public Napi::ObjectWrap<Example> {
+  ///       public:
+  ///         static void Initialize(Napi::Env& env, Napi::Object& target) {
+  ///           Napi::Function constructor = DefineClass(env, "Example", {
+  ///             InstanceAccessor("value", &GetSomething, &SetSomething),
+  ///             InstanceMethod("doSomething", &DoSomething),
+  ///           });
+  ///           target.Set("Example", constructor);
+  ///         }
+  ///
+  ///         Example(const Napi::CallbackInfo& info); // Constructor
+  ///         Napi::Value GetSomething(const Napi::CallbackInfo& info);
+  ///         void SetSomething(const Napi::CallbackInfo& info, const Napi::Value& value);
+  ///         Napi::Value DoSomething(const Napi::CallbackInfo& info);
+  ///       }
```

Comment:
Nitpicky: This brace is not aligned with the class statement - looks like it's 2 spaces further right.